src/main/webapp/destinationCreation.html

```diff
@@ -3,11 +3,18 @@
   <head>
     <meta charset="UTF-8"/>
     <title>Destination Creation</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
     <link rel="stylesheet" href="destinationCreation.css"/>
+    <script src="config.js"></script>
+    <script src="destinationCreation.js"></script>
   </head>
-  <body>
+  <body onload="addScriptToHead()">
     <div>
       <h1>Create a Scavenger Hunt Destination</h1>
+      <div>
+        <input type="text" placeholder="Search.." name="search" id="search">
+        <button type="button" onclick="searchForPlace()"><i class="fa fa-search"></i></button>
```

Comment:
What is the `<i>` tag for? Should the button have a label?

Reply:
The documentation I saw for using this class had the <i> tag, but I tried using it without the <i> tag and it works fine so I removed it. I don't think it needs a label since it is clearly part of the search component.